docs/ref/contrib/gis/tutorial.txt

```diff
@@ -452,11 +452,21 @@ with the following code::
         'mpoly' : 'MULTIPOLYGON',
     }
 
-    world_shp = os.path.abspath(os.path.join(os.path.dirname(__file__), 'data', 'TM_WORLD_BORDERS-0.3.shp'))
+    world_shp = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__),
+            'data',
+            'TM_WORLD_BORDERS-0.3.shp'),
```

Comment:
The trailing `)` here is wrong. But bottom line, I don't mind if we either stick to the current implementation or would go with
```python
world_shp = os.path.abspath(
    os.path.join(a, b, c)
)
```